implementations/rust/ockam/ockam_command/src/project/enroll.rs

```diff
@@ -109,18 +117,37 @@ impl EnrollCommand {
 
             let auth0 = OidcService::new(Arc::new(OktaOidcProvider::new(okta_config)));
             let token = auth0.get_token_interactively(&opts).await?;
-            authority_node.enroll_with_oidc_token(ctx, token).await?;
+            authority_node_client
+                .enroll_with_oidc_token(ctx, token)
+                .await?;
         };
 
         // Issue credential
-        let credential = authority_node.issue_credential(ctx).await?;
+        let credential = authority_node_client.issue_credential(ctx).await?;
+
+        // Get the project name to display to the user.
+        let project_name = {
+            let project = opts
+                .state
+                .get_project_by_name_or_default(&self.trust_opts.project_name.clone())
+                .await?;
+            project.name()
+        };
 
+        // Display success message to stderr.
+        opts.terminal.write_line(&fmt_ok!(
+            "Successfully enrolled identity to the {} project. {}. {}.",
+            color_primary(project_name),
+            "The identity has the following credential in this project",
+            "The attributes below are attested by the project's membership authority"
```

Comment:
Should we print these two messages in separate lines?

Reply:
great idea, done